### DISCUSSION TITLE
Database backend MySQL vs PostgreSQL

### DISCUSSION THREAD

**Discussion by vizeit** [apache/airflow#42240] [General]
Back in June’24, I had a discussion on Slack about a database issue. My database backend for the Airflow instance is MySQL. It was recommended to migrate to PostgreSQL to resolve such issues. I was also told that MySQL may not be supported in the future versions. I configured PostgreSQL and performed few tests to compare both the DBs with the type of heavy workload I was expecting for my airflow instance in the production environment. The test results did not show a reason to switch from MySQL to PostgreSQL. In fact, PostgreSQL performed slower and airflow configuration cost more compared to MySQL. I wanted to start this discussion to find out if others have any similar observations about Postgres and what is Airflow community planning to do about MySQL support in the future versions?

**Comment by potiuk** [comment]
IMHO MySQL is going to stay, there is no consensus on removing it. You can discuss it if you want more certain answer at the devlist - this is where such important decisions are discussed, you can also search through archives (see community section of our website) for devlist discussions on MySQL and see for yourself what's the discussion status.

**Reply by vizeit** [reply]
Thanks, I have started a discussion on devlist
